File: fc-chat/demo/part3/ch4/message-system-add-push/src/main/java/net/prostars/messagesystem/dto/websocket/inbound/KeepAlive.java
package net.prostars.messagesystem.dto.websocket.inbound;

import com.fasterxml.jackson.annotation.JsonCreator;
import net.prostars.messagesystem.constant.MessageType;

public class KeepAlive extends BaseRequest {

  @JsonCreator
  public KeepAlive() {
    super(MessageType.KEEP_ALIVE);
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/inbound/KeepAlive.kt
package com.yscorp.withpush.messagesystem.dto.websocket.inbound

import com.yscorp.withpush.messagesystem.constant.MessageType


class KeepAlive @com.fasterxml.jackson.annotation.JsonCreator constructor() : BaseRequest(MessageType.KEEP_ALIVE)
